Enyim.Caching/Memcached/MemcachedNode.cs

```diff
@@ -818,6 +947,7 @@ protected internal virtual PooledSocket CreateSocket()
             {
                 var ps = new PooledSocket(_endPoint, _config.ConnectionTimeout, _config.ReceiveTimeout, _logger);
                 ps.Connect();
+                _metricFunctions.Inc("cache_connection_count", EndPointString);

```

Comment:
We will also have to decrement this counter when the connection is removed right?